backend/app/services/simulation_manager.py

```
@@ -1,529 +1,161 @@
-    
-    def get_simulation(self, simulation_id: str) -> Optional[SimulationState]:
-        """获取模拟状态"""
-        return self._load_simulation_state(simulation_id)
-    
-    def list_simulations(self, project_id: Optional[str] = None) -> List[SimulationState]:
-        """列出所有模拟"""
+
+    async def get_simulation(self, simulation_id: str) -> Optional[SimulationState]:
+        return await self._load_simulation_state(simulation_id)
+
+    async def list_simulations(self, project_id: Optional[str] = None, limit: int = None) -> List[SimulationState]:
```

Ruff (0.15.11)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)